common/install.sh

```diff
@@ -90,16 +90,28 @@ $FUNCTION "DOWN"
 # fi
 
 ui_print " "
-ui_print " - Select Option -"
-ui_print "   Do you want overlays (theme accent and rounded corners) enabled:"
-ui_print "   Vol Up = Yes, Vol Down = No"
+ui_print " - Overlay Options -"
+ui_print "   Do you want overlays (theme accent and rounded corners) enabled?"
+ui_print "   Vol Up = Yes, Vol Down = More Options"
 if $FUNCTION; then
   ui_print " "
   ui_print "   Enabling overlays..."
 else
   ui_print " "
-  ui_print "   Disabling overlays..."
-  sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
-  rm -rf /data/resource-cache
-  ui_print "You may want to clear Dalvik cache too!"
+  ui_print " - Overlay Options -"
+  ui_print "   Do you want overlays (rounded corners) enabled?"
+  ui_print "   Vol Up = Yes, Vol Down = No"
+  
+  if $FUNCTION; then
+    ui_print " "
+    ui_print "   Enabling overlays..."
+    sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
+  else
+    ui_print " "
+    ui_print "   Disabling overlays..."
+    sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
+    rm -f $INSTALLER/system/vendor/overlay/Pix3lify.apk
```

Comment:
Shouldn't totally kill

Reply:
@thehappydinoa You sure? It's "disabling overlays" not disabling the theme so I would say that this is correct in my opinion. I think I got an idea tho.